ios/FluentUI/People Picker/ContactView.swift

```diff
@@ -0,0 +1,184 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+open class ContactView: UIView {
+    private struct Constants {
+        static let contactWidth: CGFloat = 70.0
+        static let avatarViewWidth: CGFloat = 70.0
+        static let avatarViewHeight: CGFloat = 70.0
+        static let titleLabelMinimumHeight: CGFloat = 12.0
+        static let spacingBetweenAvatarAndFirstLabel: CGFloat = 13.0
+    }
+
+    // TODO: Set it as the default image (pawn)
```

Comment:
AvatarView should take care of showing the pawn, not this control.
I'd remove that TODO comment as you would have no extra work to do in this class to support that.

Reply:
Got it. Setting the image to the pawn within `AvatarView` makes a lot more sense.